### PR TITLE
Add doc about how to debug a transaction with Godwoken RPC

### PR DESCRIPTION
**Pull request by magicalne**
How to debug tx with debug rpc. Give an example on mainnet v1.

### CODE REVIEW

**Review by jjyr — APPROVED** [review]

File: docs/debugRPC.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+When you send an Ethereum transaction to Polyjuice, it will be transformed into a Godwoken transaction and executed by the Godwoken VM. If your transaction fails, you may want to debug it and find out the root cause of the error.
+
+In this document, we will show you how to debug a failed transaction with Polyjuice and Godwoken. You will need:
+
+- A node running Polyjuice and Godwoken with debug RPC module enabled
+- A transaction hash of a failed Ethereum transaction
+- A tool that can send JSON-RPC requests, such as curl or Postman
+
+Attention! Before executing debug rpc, you have to make sure the node you connect has enabled DEBUG already. Here is the config of mainnet readonly node about how to enable DEBUG: [enable debug RPC](https://github.com/godwokenrises/godwoken-info/blob/main/mainnet_v1/gw-mainnet_v1-config-readonly.toml#L31).
```
Comment (Flouse) [inline docs/debugRPC.md:19]: wrong guide
```sh
enable_methods = ["debug"] # enable debug RPC
```

File: docs/debugRPC.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+## Steps
+
+1. Find out the transaction hash of the failed Ethereum transaction. You can use an explorer like [gwscan](https://gwscan.com) to search for your transaction by its sender address, receiver address, or just transcation hash. Alternatively, you can use your own node's RPC method `eth_getTransactionByHash` to retrieve the transaction details by its hash.
```
Comment (Flouse) [inline docs/debugRPC.md:23]:
```suggestion
1. Find out **the transaction hash** of the failed Ethereum transaction. You can use an explorer like [gwscan](https://gwscan.com) to search for your transaction by its sender address, receiver address, or just transcation hash.
```

File: docs/debugRPC.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+2. Connect to your node's JSON-RPC endpoint using curl or Postman. For example, if your node is running on localhost:8024, you can use curl as follows:
+
+```bash
+curl -X POST http://localhost:8024 -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"<method>","params":["<params>"],"id":1
+```
+
+3. Use the debug RPC method `poly_getGwTxHashByEthTxHash` to transform your Ethereum transaction hash into a Godwoken transaction hash. You need to pass the Ethereum transaction hash as the parameter. For example:
```
Comment (Flouse) [inline docs/debugRPC.md:31]:
```suggestion
3. Use the RPC method `poly_getGwTxHashByEthTxHash` to transform your Ethereum transaction hash into a Godwoken transaction hash. You need to pass the Ethereum transaction hash as the parameter. For example:
```

File: docs/debugRPC.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+In this document, we will show you how to debug a failed transaction through `gw_debug_replay_transaction` RPC. You will need:
+
+- A node running Polyjuice and Godwoken with debug RPC module enabled
+- A transaction hash of a failed Ethereum transaction
+- A tool that can send JSON-RPC requests, such as curl or Postman
+
+Attention! Before executing debug rpc, you have to make sure the node you connect has enabled DEBUG already. If you are running a devnet in your local environment, you can enable debug RPC by updating your `config.toml` to:
+```toml
+[rpc_server]
+listen = '0.0.0.0:8119'
```
Comment (Flouse) [inline docs/debugRPC.md:22]:
```suggestion
```

**Review by Flouse — APPROVED** [review]

**Pull request merged**